pkg/gui/custom_commands.go

```diff
@@ -146,13 +156,26 @@ func (gui *Gui) GenerateMenuCandidates(commandOutput string, filter string, form
 				}
 			}
 		}
-		err = temp.Execute(buff, tmplData)
+		err = tempItem.Execute(buffItem, tmplData)
+		if err != nil {
+			return candidates, gui.surfaceError(err)
+		}
+		err = tempDescr.Execute(buffDescr, tmplData)
 		if err != nil {
 			return candidates, gui.surfaceError(err)
 		}
 
-		candidates = append(candidates, strings.TrimSpace(buff.String()))
-		buff.Reset()
+		// Populate menu entry
+		// label formatted as labelFormat
+		// value as valueFormat
+		entry := CommandMenuEntry{
+			strings.TrimSpace(buffDescr.String()),
+			//"Description",
+			strings.TrimSpace(buffItem.String()),
+		}
```

Comment:
I don't think we need all the comments, this explains it all :)
```suggestion
		entry := CommandMenuEntry{
			label: strings.TrimSpace(buffDescr.String()),
			value: strings.TrimSpace(buffItem.String()),
		}
```